src/revolve/db/postgres_adapter.py

```
@@ -0,0 +1,677 @@
+        LEFT JOIN pg_type pt ON pt.typname = c.udt_name
+        WHERE c.table_schema NOT IN ('pg_catalog', 'information_schema')
+        GROUP BY c.table_name
+    ) AS sub
+    """)
+
+        return query_result
+
+
+    def order_tables_by_dependencies(self, dependencies: Dict[str, Any]) -> List[str]:
+        # Extract child tables and all referenced parent tables
+        child_tables = set(dependencies)
```

Ruff (0.11.9)

Local variable `child_tables` is assigned to but never used

Remove assignment to unused variable `child_tables`

(F841)